UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.API.FunctionalTests/Helper/FileContentHelper.cs

```diff
@@ -15,7 +15,7 @@ public static class FileContentHelper
         private const string FileContent_avcs = "AVCS";
         private const string FileContent_base = "Base";
         private const string FileContent_dvd = "Media','DVD_SERVICE'";
-
+        public static string dirName, baseFolderNumber;
```

Comment:
Are these used? If not remove them please.

Reply:
Updated. Thanks